helpers/notification.js

```diff
@@ -0,0 +1,33 @@
+module.exports = function(subject, type, date, file, path, id) {
+  var nodemailer = require('nodemailer');
+  var smtpTransport = require('nodemailer-smtp-transport');
+  var markdown = require('nodemailer-markdown').markdown;
+
+  var transporter = nodemailer.createTransport(smtpTransport({
+      service: 'gmail',
+      auth: {
+          user: process.env.NODEMAILER_USER,
```

Comment:
Hmm, I think it's better if this is just called `SMTP_USER` and `SMTP_PASSWORD`. Makes more sense to just associated it with SMTP.